frontend/src/metabase/collections/components/CollectionCopyEntityModal.tsx

```diff
@@ -38,34 +34,20 @@ function CollectionCopyEntityModal({
   initialCollectionId,
   onClose,
   onSaved,
-  triggerToast,
+}: {
+  entityObject: any;
+  initialCollectionId: number;
+  onClose: () => void;
+  onSaved: (newEntityObject: any) => void;
 }) {
   const [isShallowCopy, setIsShallowCopy] = useState(true);
   const title = getTitle(entityObject, isShallowCopy);
 
-  const handleValuesChange = ({ is_shallow_copy }) => {
+  const handleValuesChange = ({ is_shallow_copy }: Record<string, any>) => {
     setIsShallowCopy(is_shallow_copy);
   };
 
-  const handleSaved = newEntityObject => {
-    const newEntityUrl = Urls.modelToUrl({
-      model: entityObject.model,
-      model_object: newEntityObject,
-    });
-
-    triggerToast(
-      <div className={cx(CS.flex, CS.alignCenter)}>
-        {/* A shallow-copied newEntityObject will not include `uncopied` */}
-        {newEntityObject.uncopied?.length > 0
-          ? t`Duplicated ${entityObject.model}, but couldn't duplicate some questions`
-          : t`Duplicated ${entityObject.model}`}
-        <Link className={cx(CS.link, CS.textBold, CS.ml1)} to={newEntityUrl}>
-          {t`See it`}
-        </Link>
-      </div>,
-      { icon: entityObject.model },
-    );
```

Comment:
Nothing invokes this component with the triggerToast prop set to anything, so this seems to be dead code